packages/workflow/src/documents.ts

```diff
@@ -70,57 +70,70 @@ export async function uploadCertificateAttachmentsToDocumentsStore<
   return certificateDetails
 }
 
+function isPresignedUrl(url: string) {
+  return url.includes(MINIO_URL)
```

Comment:
Can you just test for `startsWith('http')` or something like that? Conceptually the workflow service (or other services than the documents service) shouldn't know Minio exists.